src/main/java/com/coronacarecard/service/impl/AWSS3ServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+package com.coronacarecard.service.impl;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.*;
+import com.coronacarecard.exceptions.InternalException;
+import com.coronacarecard.service.AWSS3Service;
+import org.springframework.stereotype.Service;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+
+@Service
+public class AWSS3ServiceImpl implements AWSS3Service {
+
+    private static final String AWS_ACCESS_KEY        = "AKIAQECDVEZ5V4CES3KO";
```

Comment:
Add a to-do to remove them we don't want this here also if u are issuing creds I gave u remember it has admin right remove that and create new creds with just access to s3